osu.Game/Screens/SelectV2/BeatmapCarousel.cs

```diff
@@ -98,7 +100,7 @@ public BeatmapCarousel()
             {
                 new BeatmapCarouselFilterMatching(() => Criteria!),
                 new BeatmapCarouselFilterSorting(() => Criteria!),
-                grouping = new BeatmapCarouselFilterGrouping(() => Criteria!, () => detachedCollections())
+                grouping = new BeatmapCarouselFilterGrouping(() => Criteria!, () => realm)
```

Comment:
why was this moved?

Reply:
Since I'm including realm directly, I thought there's not really any reason to still have the `detachedCollections` func. If you don't think that's the correct approach and the `realm` func will eventually be removed, then I'll revert it.